hardware_pydantic/devices.py

```diff
@@ -4,34 +4,83 @@
 
 
 class Heater(Device):
```

Comment:
for Junior heat/cool/stir are all done on a rack slot, I would add these to [here](https://github.com/qai222/simulator-aspire-poc/blob/ca53d1a7e557080fa8c34931563e0bb6f768b654/hardware_pydantic/junior/junior_devices.py#LL18C10-L18C10)